src/wp-admin/about.php

```diff
@@ -55,162 +55,183 @@
 					?>
 				</h2>
 				<p class="is-subheading">
-					<?php
-					printf(
-						/* translators: 1: Count of enhancements, 2: Count of bug fixes. */
-						__( 'WordPress 6.2 includes more than %1$s enhancements and %2$s bug fixes. This page highlights the latest features since the November 2022 release of WordPress 6.1. From quick highlights to developer resources, there&#8217;s a lot to explore.' ),
-						292, // Enhancements.
-						394 // Bug fixes.
-					);
-					?>
+					<?php _e( 'Create beautiful and compelling websites more efficiently than ever. Whether you want to build an entire site without coding or are a developer looking to customize every detail, WordPress 6.3 has something for you.' ); ?>
 				</p>
 			</div>
 		</div>
 
 		<div class="about__section has-2-columns">
 			<div class="column">
 				<div class="about__image">
-					<img src="https://s.w.org/images/core/6.2/about-site-editor.png" alt="" />
+					<img src="data:image/svg+xml,%3Csvg width='436' height='436' viewbox='0 0 100 100' xmlns='http://www.w3.org/2000/svg'%3E%3Crect width='100%25' height='100%25' fill='%23dcdcde' /%3E%3C/svg%3E" alt="" />
 				</div>
 			</div>
 			<div class="column is-vertically-aligned-center">
-				<h3><?php _e( 'Explore and edit your site from the Site Editor' ); ?></h3>
-				<p><?php _e( 'An updated interface gives you more control over your site editing experience. Browse through full previews of your templates and template parts, then jump into editing your site from wherever you choose.' ); ?></p>
+				<h3><?php _e( 'Do everything in the Site Editor' ); ?></h3>
+				<p><?php _e( 'WordPress 6.3 brings your content, templates, and patterns together in the Site Editor for the first time. Add pages, browse style variations, create synced patterns, and enjoy fine-tuned control over your navigation menus. No more time wasted switching across different site areas—now you can focus on what matters most. Creation to completion, all in one place.' ); ?></p>
 			</div>
 		</div>
 
 		<div class="about__section has-2-columns">
 			<div class="column is-vertically-aligned-center">
-				<h3><?php _e( 'Manage your menu in more ways with the Navigation block' ); ?></h3>
-				<p><?php _e( 'A new sidebar experience makes it easier to edit your site&#8217;s navigation. Add, remove, and reorder menu items faster—no matter how complex your menus are.' ); ?></p>
+				<h3><?php _e( 'Create and sync patterns' ); ?></h3>
+				<p><?php _e( 'Arrange Blocks in unlimited ways and save them as Patterns for use throughout your site. You can even specify whether to sync your patterns (previously referred to as “reusable blocks”) so that one change applies to all parts of your site. Or, utilize patterns as a starting point with the ability to customize each instance.' ); ?></p>
```

Comment:
"Blocks" should not be capitalized.

Reply:
Done:

<img width="495" alt="Screenshot 2023-07-18 at 11 11 32 AM" src="https://github.com/WordPress/wordpress-develop/assets/541093/0d7e88b9-5ecf-4106-a4fd-3a6268e1146e">